docs/README.rst

```diff
@@ -1,16 +1,16 @@
-======
```

Comment:
I'm confused why this file exists. You should just be able to include the top-level readme.

Also, you don't need to convert everything to RST, you can just use the `myst` package to do the conversion dynamically.